sklearn/tests/test_calibration.py

```diff
@@ -578,6 +583,22 @@ def test_calibration_attributes(clf, cv):
         assert calib_clf.n_features_in_ == X.shape[1]
 
 
+def test_calibration_inconsistent_prefit_n_features_in():
+    # Check that `n_features_in_` from prefit base estimator
+    # is consistent with training set
+    X, y = make_classification(n_samples=10, n_features=5,
+                               n_classes=2, random_state=7)
+    clf = LinearSVC(C=1).fit(X, y)
+    calib_clf = CalibratedClassifierCV(clf, cv='prefit')
+
+    msg = re.escape(
+        "Base estimator LinearSVC was prefit on 5 features "
+        "but CalibratedClassifierCV is fit with 3 features."
+    )
```

Comment:
curious, which part of this message needs the `re.escape`?

Reply:
I found it cleaner not to interpret the final `.` as a regexp pattern. But it would also work otherwise.

Reply:
Simplified.